torch/csrc/distributed/c10d/ProcessGroupNCCL.cpp

```diff
@@ -1113,6 +1113,59 @@ bool ProcessGroupNCCL::isInitialized() {
   return initialized;
 }
 
+void ProcessGroupNCCL::registerMemPool(c10::cuda::MemPool* pool) {
+  const auto key = std::to_string(pool->device());
+  auto device = at::Device(at::DeviceType::CUDA, pool->device());
+  LOG(INFO) << logPrefix()
```

Comment:
Is this meant to be left in?

Reply:
I think it's ok to leave this as it will be good for debugging.